lib/atom-ternjs-helper.js

```diff
@@ -30,6 +30,13 @@ export function isValidEditor(editor) {
     return false;
   }
 
+  const scopeChain = editor.getRootScopeDescriptor().getScopeChain();
+
+  if (/^\.source\.js\b/.test(scopeChain)) {
```

Comment:
If you're looking to also target jsx files this would work:
```suggestion
  if (/^\.source\.jsx?\b/.test(scopeChain)) {
```